gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/model/StoreType.java

```diff
@@ -13,18 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package uk.gov.gchq.gaffer.gaas.model;
-
-public class GraphSpec {
 
-    private NewGraph graph;
-
-    public GraphSpec graph(final NewGraph graph) {
-        this.graph = graph;
-        return this;
-    }
+package uk.gov.gchq.gaffer.gaas.model;
 
-    public NewGraph getGraph() {
-        return graph;
-    }
+public enum StoreType {
+    // TODO: rename to ACCUMULO_STORE & MAP_STORE
```

Comment:
No longer needed. Should be a string instead.

Reply:
Deleted